ci/checksizes.sh

```diff
@@ -34,22 +34,22 @@ REVERSI_OPT_GZ_SIZE=$(stat '-c%s' "$REVERSI_OPT.gz")
 
 case $FULLVER in
   2.10.2)
-    REVERSI_PREOPT_EXPECTEDSIZE=662000
-    REVERSI_OPT_EXPECTEDSIZE=143000
-    REVERSI_PREOPT_GZ_EXPECTEDSIZE=89000
-    REVERSI_OPT_GZ_EXPECTEDSIZE=40000
+    REVERSI_PREOPT_EXPECTEDSIZE=698000
+    REVERSI_OPT_EXPECTEDSIZE=165000
+    REVERSI_PREOPT_GZ_EXPECTEDSIZE=99000
+    REVERSI_OPT_GZ_EXPECTEDSIZE=49000
     ;;
   2.11.7)
-    REVERSI_PREOPT_EXPECTEDSIZE=608000
-    REVERSI_OPT_EXPECTEDSIZE=134000
-    REVERSI_PREOPT_GZ_EXPECTEDSIZE=84000
-    REVERSI_OPT_GZ_EXPECTEDSIZE=38000
+    REVERSI_PREOPT_EXPECTEDSIZE=639000
+    REVERSI_OPT_EXPECTEDSIZE=155000
```

Comment:
Did not increase because the character count of  `charTypeIndices` was reduced by half by representing it with deltas.